spec/keytar-spec.js

```diff
@@ -0,0 +1,130 @@
+var assert = require('chai').assert
```

Comment:
I thought about it but it seemed a pretty hefty price for a small test suite. If you think it'd be worth it I'm happy to pull it in.